Description: Store last_known_working.quirkdb in /var/cache
 last_known_working.quirkdb is more of a cache file then a configuration file
 and e.g. changes on kernel upgrades, so we don't want it in /etc/pm.
 Move it to /var/cache/pm-utils instead
Author: Michael Biebl <biebl@debian.org>
Index: pm-utils/pm/pm-functions.in
===================================================================
--- pm-utils.orig/pm/pm-functions.in	2010-03-18 01:44:33.000000000 +0100
+++ pm-utils/pm/pm-functions.in	2010-03-18 01:45:41.206963690 +0100
@@ -28,7 +28,7 @@
 DX=252
 PM_FUNCTIONS="$PM_UTILS_LIBDIR/functions"
 PM_QUIRKDB="$PM_UTILS_LIBDIR/video-quirks"
-PM_LKW_QUIRKS="$PM_UTILS_ETCDIR/last_known_working.quirkdb"
+PM_LKW_QUIRKS="/var/cache/pm-utils/last_known_working.quirkdb"
 # Use c sort order
 LC_COLLATE=C
 
Index: pm-utils/man/pm-action.xml
===================================================================
--- pm-utils.orig/man/pm-action.xml	2010-03-18 01:44:28.000000000 +0100
+++ pm-utils/man/pm-action.xml	2010-03-18 01:45:41.206963690 +0100
@@ -285,12 +285,13 @@
 	    <command>pm-suspend</command> or 
 	    <command>pm-suspend-hybrid</command> as an .quirkdb file that 
 	    is specific to this system.  The file will be saved in 
-	    <filename>/etc/pm/last_known_working.quirkdb</filename>.
-	    This parameter will only save the actual quirks that were 
-	    used to successfully suspend/resume a system, and will be 
+	    <filename>/var/cache/pm-utils/last_known_working.quirkdb</filename>.
+	    This parameter will only save the actual quirks that were
+	    used to successfully suspend/resume a system, and will be
 	    specific to the exact configuration of that system, including
 	    the video hardware, video driver, and whether or not kernel
-	    modesetting was used.
+	    modesetting was used. If the system configuration changes, like after
+	    a kernel upgrade, this file will be overwritten.
           </para>
         </listitem>
       </varlistentry>
